operator/cmd/mesh/install.go

```diff
@@ -186,9 +186,6 @@ func Install(kubeClient kube.CLIClient, rootArgs *RootArgs, iArgs *InstallArgs,
 	if !rootArgs.DryRun && !iArgs.SkipConfirmation {
 		prompt := fmt.Sprintf("This will install the Istio %s %q profile (with components: %s) into the cluster. Proceed? (y/N)",
 			tag, profile, humanReadableJoin(enabledComponents))
-		if profile == "empty" {
```

Comment:
Maybe it was really meant to be `if len(enabledComponents) == 0`... although I don't know why you install in that case :-)